spec/api-clipboard-spec.js

```diff
@@ -1,10 +1,12 @@
 const { expect } = require('chai');
 const path = require('path');
 const { Buffer } = require('buffer');
+const { ifdescribe } = require('./spec-helpers');
 
 const { clipboard, nativeImage } = require('electron');
 
-describe('clipboard module', () => {
+// FIXME(zcbenz): Clipboard tests are failing on WOA.
+ifdescribe(process.platform !== 'win32' || process.arch !== 'arm64')('clipboard module', () => {
```

Comment:
@jkleinsc what is the build number of windows on our testing ci ? I was unable to repro the failures on my local device with Build 21390.1000

Reply:
@deepak1556 I can check tomorrow, but I think the issue only happens when running through CI but if run those tests outside of CI on the same machine they pass.

Reply:
Ah okay, nevermind then. Thanks!